### ISSUE TITLE
Button disappearing in Safari Webview

### ISSUE THREAD

**Issue by timi2shoes** [HubSpot/vex#210]
I found a bug in Safari Webview where when a vex-dialog-button is focused and then a drop down is selected the button disappears. I was able to fix the problem by removing `-webkit-backface-visibility: hidden;` from `vex-theme-top.css line 404`.

[https://github.com/HubSpot/vex/blob/3544c7386aba85d91b0e68d398a0968e3d8d57c6/dist/css/vex-theme-top.css#L404](url)

My question though is why is `-webkit-backface-visibility: hidden;` used there since there does not appear from what i can see a need for it there?

**Comment by bbatliner** [comment]
You're right, there doesn't seem to be a need for the `backface-visibility`.

Can you include a picture/gif of the original problem, and your fix? At that point I will be comfortable merging a PR in to fix this.

Thank you!

**Comment by timi2shoes** [comment]
So what happens is that the button always stays there, it can be clicked, but it disappears and cannot be seen.

![safari-focus-click-drop-down-button-visible](https://cloud.githubusercontent.com/assets/11464681/23808449/8844b398-058f-11e7-8f0d-77eb94448127.png)


![safari-focus-click-drop-down-button-disappearing](https://cloud.githubusercontent.com/assets/11464681/23808391/4b6ac0a2-058f-11e7-8d92-8a3fa30504b0.png)

In my fix I removed `-webkit-backface-visibility: hidden;` from line 404 and 413. I did notice that `-webkit-backface-visibility: hidden;` was used line 297 and 305, not sure if those are required in there too.
